### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots/code**
+If applicable, add screenshots/code samples to help explain your problem.
+
+**Versions (please complete the following information):**
+ - Lucky version:
+ - OS:
```
Comment (jwoertink) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: Maybe also crystal version?

**Review by jwoertink — COMMENTED** [review]
What happens if it's not a bug report?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (paulcsmith) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: Good idea!

**Comment by paulcsmith** [comment]
If its not a bug report they will still have the other buttons to make a feature request, discuss in forums, or in chat

**Review by jwoertink — APPROVED** [review]

**Comment by jwoertink** [comment]
Awesome! I like it 👍

**Comment by paulcsmith** [comment]
Thanks for the review!

**Pull request merged**